newswires/client/src/WireDetail.tsx

```diff
@@ -52,13 +53,15 @@ export const WireDetail = ({ wire }: { wire: WireData }) => {
 
 	return (
 		<Fragment>
-			<h3
-				css={css`
-					font-weight: 300;
-				`}
-			>
-				{wire.content.subhead}
-			</h3>
+			<EuiScreenReaderLive focusRegionOnTextChange>
```

Comment:
this means that the Item box gets focused every time a new item is selected, which feels like the right behaviour for screenreaders and for general keyboard support?

Reply:
Sounds right for screenreaders, but I think I could argue the opposite for "general" keyboard support - if I was keyboard controlling I think most handy would be a way to "peek" into stories as I scroll through them without moving my focus, so I could then quickly move onto the next in the story. Might be worthy of more nuanced keyboard control than just tab-support though, so possibly a future improvement

Reply:
> if I was keyboard controlling I think most handy would be a way to "peek" into stories as I scroll through them without moving my focus, so I could then quickly move onto the next in the story. Might be worthy of more nuanced keyboard control than just tab-support though, so possibly a future improvement

I think I basically agree with you. Initially I was thinking that it was okay because we'd encourage using arrow keys for switching between items, rather than tab keys. However I've been rethinking that a bit. But agree that we probably want to have a more 'holistic' review of the keyboard behaviour later once we've got a clearer sense of the overall shape of the app?